models/booking.js

```diff
@@ -1,7 +1,20 @@
 const mongoose = require('mongoose');
 
 const bookingSchema = new mongoose.Schema({
-
+  author: {
+    id: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'user'
+    },
+    username: String,
+    required: true
+  },
+  startDate: {type: Date, required: true},
+  endDate: {type: Date, required: true},
+  price: {type: Number, required: true, min: 0.01},
+  origin: {type: String, required: true},
+  guests: {type: Number, required: true, min: 1},
+  nation: {type: String}
```

Comment:
Are you sure your eslint is on? Object parenthesis always need to have space inside, i.e.:
```js
// Bad
const object = {prop: 'prop'};
// Good
const object = { prop: 'prop' };
```